### PR TITLE
Feat/api conection authentication

### PR DESCRIPTION
**Pull request by elmejorlui**
Fixed the cookies

### CODE REVIEW

File: apps/siiges-app/pages/usuarios/utils/deleteUser.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default function deleteUser(id) {
+  fetch(`http://localhost:3000/api/v1/usuarios/${id}`, {
+    method: 'DELETE',
```
Comment (hecttoressau) [inline apps/siiges-app/pages/usuarios/utils/deleteUser.jsx:3]: Did this work? Yo don't need to use headers?

**Review by hecttoressau — COMMENTED** [review]
I left a question, depending on that I'll approve it

**Review by hecttoressau — APPROVED** [review]
LGTM

**Review by csiges — APPROVED** [review]
LGTM

**Pull request merged**